addons/mail/static/src/js/emojis_mixin.js

```diff
@@ -8,7 +8,7 @@ import { escape } from "@web/core/utils/strings";
  * It's currently used to format text and wrap the emojis around <span class="o_mail_emoji"> to make them look nicer
  *
  */
-export default {
```

Comment:
we can just get rid of the mixin entirely since it now only contains a stateless utility function